source/connect-to-mongo/stable-api.txt

```diff
@@ -0,0 +1,134 @@
+.. _java-rs-stable-api:
+
+==========
+{+stable-api+}
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatible, backwards, upgrade
+
+.. note::
+
+   The {+stable-api+} feature requires {+mdb-server+} 5.0 or later.
+
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. When you update either your driver or server, 
+the API version changes, which can change the way these operations behave.
```

Comment:
> When you update either your driver or server,  the API version changes, which can change the way these operations behave.

This is not accurate.  Was this text copied from somewhere else?

https://www.mongodb.com/docs/manual/reference/stable-api/ looks accurate to me, so we should either link to it or crib from it.

Reply:
This description was taken from the same page from a different driver, yes. Can take this back to the team to investigate whether there's a larger-scope misunderstanding here.